src/components/Modal/Modal.tsx

```diff
@@ -1,50 +1,69 @@
 import styles from './Modal.module.scss';
-import { PropsWithChildren } from 'react';
+import { PropsWithChildren, useState } from 'react';
 
 interface ModalProps {
   h3: string;
-  onClose: () => void;
-  onCanсel: () => void;
   onSave: () => void;
+  nameevent: string;
 }
 
 export default function Modal(props: PropsWithChildren<ModalProps>) {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const handleButtonClick = () => {
+    setModalOpen(true);
+  };
```

Comment:
if used once, maybe inline? i.e. no need in a separate function here, just move as () => setModalOpen(true) to the single place it's used

Reply:
agree